Themes/default/languages/index.english.php

```diff
@@ -243,7 +243,6 @@
 // Use numeric entities in the below string.
 $txt['hello_guest'] = 'Welcome,';
 $txt['welmsg_hey'] = 'Hey,';
-$txt['welmsg_welcome'] = 'Welcome,';
```

Comment:
Did you find a place where the other two are used? My issue was about all three `welmsg_*` strings.

Reply:
I addressed this during the rebase.